### ISSUE TITLE
Music abruptly stops during game startup, is fine when returning from a world

### ISSUE THREAD

**Issue by Cervator** [MovingBlocks/Terasology#2364]
I suspect there is a context change of some sort with the music starting to play in one then gets killed off before the menu loads fully. As such might relate a little to #2074 

Disappears between these two statements:

```
14:14:02.011 [main] WARN  o.t.rendering.nui.skin.UISkinFormat - Failed to resolve UIWidget class UIInputBindButton, skipping style information
14:14:03.038 [main] WARN  o.t.engine.internal.TimeBase - Delta too great (16124), capping to 1000
```

When returning to the main menu from a world the menu theme starts and keeps playing as expected, it only stops abruptly on initial game load.
##

**Comment by Cervator** [comment]
Interestingly this seems specific to running the game from source, at least in my mega workspace. If I run the Alpha 4 binary the music continues playing in the main menu.

**Comment by Cervator** [comment]
Additional bit of detail: this *may* be instead about letting the game start in the background (which is the case at least for me running from IntelliJ in Windows 8 on my tablet). If I grab the game window before it fully initializes and it loads while the active window the menu music keeps playing after initial startup just fine.

**Comment by Cervator** [comment]
Closing as possibly obsolete or fixed by #2945 - we can always reopen later. Likely some needed context/environment tweaks will effectively reset this anyway sooner or later.
